src/scss/helpers/_mq.scss

```diff
@@ -47,24 +69,24 @@
   @return $number;
 }
 
-@mixin bp-suffix($class, $original: true, $breakpoints: $grid-bp, $create-between-breakpoints: false) {
+@mixin bp-suffix($class, $original: true, $breakpoints: $grid-bp, $createBetweenBreakpoints: false) {
```

Comment:
Changing the variables from hyphenated to pascal case will annoy the sass linter / codacy

Reply:
Not sure who did this and why?

Reply:
Did you re copy the file from the old pattern library?

Reply:
Ah yes, there were some bits missing. I possibly got confused on which convention was being used in v2

Reply:
I can't see anything thats been added that isn't necessary (i.e. the fixed and IE mixins) so could you revert the file bak? Theres a chance that some files using the bp-suffix will be broken now because they'll be passing `$create-between-breakpoints` rather than `$createBetweenBreakpoints`

Reply:
OK I've reverted it.